tasks/main.yml

```diff
@@ -2,51 +2,65 @@
 # make sure that the Oracle instant client is installed
 
 - name: install unzip
-  apt: name=unzip state=present update_cache=yes cache_valid_time=3600
+  apt:
+    name: unzip
+    state: present
+    update_cache: yes
+    cache_valid_time: "3600"
 
 - name: install libaio1
-  apt: name=libaio1 state=present update_cache=yes cache_valid_time=3600
+  apt:
+    name: libaio1
+    state: present
+    update_cache: yes
+    cache_valid_time: "3600"
 
 - name: install libaio-dev
-  apt: name=libaio-dev state=present update_cache=yes cache_valid_time=3600
+  apt:
+    name: libaio-dev
+    state: present
+    update_cache: yes
+    cache_valid_time: "3600"
 
 - name: make sure that the /opt/oracle directory exists
-  file: path=/opt/oracle state=directory
+  file:
+    path: /opt/oracle
+    state: directory
 
 - name: install the basiclite archive
   unarchive:
-    src: oracle/instantclient-basiclite-linux.x64-11.2.0.3.0.zip
+    src: oracle/instantclient-basiclite-linux.x64-19.6.0.0.0dbru.zip
     dest: /opt/oracle
-    creates: /opt/oracle/instantclient_11_2/BASIC_LITE_README
+    creates: /opt/oracle/instantclient_19_6/BASIC_LITE_README
 
-- name: install the sqlplus archive
+- name: install the sdk archive
   unarchive:
-    src: oracle/instantclient-sqlplus-linux.x64-11.2.0.3.0.zip
+    src: oracle/instantclient-sdk-linux.x64-19.6.0.0.0dbru.zip
     dest: /opt/oracle
-    creates: /opt/oracle/instantclient_11_2/sqlplus
+    creates: /opt/oracle/instantclient_19_6/sdk
 
-- name: install the sqlplus archive
-  unarchive: src=oracle/instantclient-sdk-linux.x64-11.2.0.3.0.zip dest=/opt/oracle creates=/opt/oracle/instantclient_11_2/sdk
+- name: copy ld.so.conf.d config file
+  copy:
+    src: ld.so.conf.d/oracle-instantclient.conf
+    dest: /etc/ld.so.conf.d
 
-- name: setup missing libclntsh symlink
-  file: src=/opt/oracle/instantclient_11_2/libclntsh.so.11.1 dest=/opt/oracle/instantclient_11_2/libclntsh.so state=link 
-
-- name: setup missing libocci symlink
-  file: src=/opt/oracle/instantclient_11_2/libocci.so.11.1 dest=/opt/oracle/instantclient_11_2/libocci.so state=link 
+- name: run ldconfig
+  command: ldconfig
 
 - name: setup oracle_home and ld_library_path in profile.d
-  copy: src=oracle_home.sh dest=/etc/profile.d/oracle_home.sh mode=755
+  copy:
+    src: oracle_home.sh
+    dest: /etc/profile.d/oracle_home.sh
+    mode: "0755"
```

Comment:
Should this mode be 755  (per line 38 of original code) or 0755?

Reply:
Actually no; it always should have been `0755`. Turns out that it wasn't working properly before...